src/metabase/pulse.clj

```diff
@@ -316,8 +344,11 @@
   [pulse results {{channel-id :channel} :details}]
   (log/debug (u/format-color 'cyan (trs "Sending Alert ({0}: {1}) via Slack" (:id pulse) (:name pulse))))
   {:channel-id  channel-id
-   :message     (trs "Alert: {0}" (first-question-name pulse))
-   :attachments (create-slack-attachment-data results)})
+   :attachments (cons {:blocks [{:type "header"
+                                 :text {:type "plain_text"
+                                        :text (str "🔔 " (first-question-name pulse))
```

Comment:
Would it be preferable to use the Unicode point for this (`U+1F514`) instead of having the emoji directly in source?  I don't have a strong preference, just wanted to make sure this wasn't something that would cause trouble.

Reply:
I don't have a strong preference either, but not sure why this would be an issue? We have emojis defined in the source already: https://github.com/metabase/metabase/blob/8afbdbb03c31061d63b7cac7fc7ddd416779f8d8/src/metabase/sync/util.clj#L203-L216

Reply:
the only reason it could be an issue would be if anyone's editor wasn't assuming UTF-8 encoding or didn't save the file as UTF-8. Writing out the code point literals would eliminate that problem, but it would also mean you couldn't actually see what you're looking at in the editor.

We haven't had any non-UTF-8 issues in the 6 years we've had those emoji characters in the code yet so I think it's something we should worry about too much. I like being able to see them personally